src/EventListener/IpTraceListener.php

```diff
@@ -33,7 +33,10 @@ public function onKernelRequest(RequestEvent $event): void
         }
     }
 
-    public static function getSubscribedEvents(): array
```

Comment:
Please don't remove native return types that were already present (i.e. revert changes in this file)